### PR TITLE
CM-1037: Support EID translation for TDD id in v7

### PR DESCRIPTION
**Pull request by wi101**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes

- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

### CODE REVIEW

File: modules/liveIntentIdSystem.js [hunk 1]
```diff
@@ -217,6 +217,10 @@ export const liveIntentIdSubmodule = {
         result.sovrn = { 'id': value.sovrn }
       }
 
+      if (value.thetradedesk) {
+        result.thetradedesk = { 'id': value.thetradedesk }
+      }
+
```
Comment (3link) [inline modules/liveIntentIdSystem.js:223]: I submitted a PR to also expose the ext.provider in v7 now: https://github.com/prebid/Prebid.js/pull/10843/files#diff-489289275f4ee5264024b54c9c851e6c57684fd2d0bccab91d8b9ccef150e235R198. We should do it here too.

File: test/spec/modules/liveIntentIdSystem_spec.js [hunk 2]
```diff
@@ -388,6 +388,11 @@ describe('LiveIntentId', function() {
     expect(result).to.eql({'lipb': {'lipbid': 'foo', 'nonId': 'foo', 'sovrn': 'bar'}, 'sovrn': {'id': 'bar'}});
   });
 
+  it('should decode a thetradedesk id to a seperate object when present', function() {
+    const result = liveIntentIdSubmodule.decode({ nonId: 'foo', thetradedesk: 'bar' });
+    expect(result).to.eql({'lipb': {'lipbid': 'foo', 'nonId': 'foo', 'thetradedesk': 'bar'}, 'thetradedesk': {'id': 'bar'}});
+  });
+
```
Comment (3link) [inline test/spec/modules/liveIntentIdSystem_spec.js:395]: What about the `test/spec/modules/liveIntentIdMinimalSystem_spec.js`?

**Review by 3link — CHANGES_REQUESTED** [review]
Please see comments.

**Review by 3link — APPROVED** [review]

**Review by peixunzhang — APPROVED** [review]

**Comment by wi101** [comment]
This change is unnecessary as the v7 won't be released anymore

**Pull request closed without merging**